.github/workflows/main.yml

```diff
@@ -20,9 +20,7 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - uses: pnpm/action-setup@v3
-        with:
-          version: 8
```

Comment:
I removed the version here and in other workflows because this action will just read from the `packageManager` field in `package.json`, so no need to duplicate this.